### PR TITLE
update to wxWidgets-2.9.3

### PR DESCRIPTION
**Pull request by karolherbst**
everything should work

### CODE REVIEW

File: src/shared/uicore/code/UICoreMain.cpp [hunk 1]
```diff
@@ -252,12 +252,12 @@ class gcAppTraits : public wxAppTraits
...
 	virtual void ScheduleForDestroy(wxObject* o)
 	{
-		m_pOldTraits->ScheduleForDestroy(o);
+		//m_pOldTraits->ScheduleForDestroy(o);
 	}
 	
 	virtual void RemoveFromPendingDelete(wxObject* o)
 	{
-		m_pOldTraits->RemoveFromPendingDelete(o);
+		//m_pOldTraits->RemoveFromPendingDelete(o);
```
Comment (karolherbst) [inline src/shared/uicore/code/UICoreMain.cpp]: lodle, what do these methods do?

ScheduleForDestroy and RemoveFromPendingDelete were private methods in wxWidgets 2.9.0 and they don't exist anymore

**Comment by karolherbst** [comment]
updated to 2.9.2

**Comment by karolherbst** [comment]
updated to 2.9.3

**Comment by Jookia** [comment]
Tell me when this works.

**Pull request merged**